src/LendingPool.sol

```diff
@@ -503,6 +503,26 @@ contract LendingPool is LendingPoolGuardian, TrustedCreditor, DebtToken, Interes
         emit Borrow(account, msg.sender, to, amount, amountWithFee - amount, referrer);
     }
 
+    /**
+     * @notice Repay a portion of a user's debt.
+     * @dev This internal function allows the caller to repay a specified amount of debt for a user.
+     * @param amount The amount to be repaid.
+     * @param account The address of the user whose debt is being repaid.
+     * @param from The address of the caller performing the repayment.
+     * @dev This function transfers tokens from the `from` address to the contract and updates the user's account balance accordingly.
+     * @dev This function does not impose access control restrictions, so it should only be called by trusted contracts and functions.
+     * @notice This function is used to manage debt repayments within the contract's internal logic.
+     * @dev Emits a `Repay` event to log the repayment details.
+     */
+    function _repay(uint256 amount, uint256 shares, address account, address from) internal {
```

Comment:
Detail, but move the internal below the two extrenal functions:
https://docs.soliditylang.org/en/latest/style-guide.html#order-of-functions